dbms/programs/server/MySQLHandler.h

```diff
@@ -77,4 +79,26 @@ class MySQLHandlerSSL : public MySQLHandler
 };
 #endif
 
+const String show_table_status_replacement_query("SELECT"
```

Comment:
It was fine to have a definition in .cpp, so that the header file is short.
But forward declaration was required to use this variable:
`const String show_table_status_replacement_query;`
since the definition was at the end of file.

But it would be even better to make this variable static and to put both declaration and definition to MySQLHandler.cpp. And the same for isFederatedServerSetupCommand:
```
static bool isFederatedServerSetupCommand(const String &query);
static const String show_table_status_replacement_query;

comQuery()
{
}

...

static bool isFederatedServerSetupCommand(const String &query)
{
}

static const String show_table_status_replacement_query("SELECT"...
```
Motivation in having a separate declaration and definition: handler code will still be compact and will be located in the beginning of file.

Reply:
@yurriy if I try to do as you suggested - move both declaration and definition to MySQLHandler.cpp (see https://gist.github.com/MaxFedotov/8ae6da39cd5c45c3d3fcaf358181298a), i've got the following error during compilation:
`
../dbms/programs/server/MySQLHandler.cpp:366:21: error: redefinition of 'const String DB::show_table_status_replacement_query'
  366 | static const String show_table_status_replacement_query("SELECT"
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../dbms/programs/server/MySQLHandler.cpp:271:21: note: 'const String DB::show_table_status_replacement_query' previously declared here
  271 | static const String show_table_status_replacement_query;
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
`

Reply:
Sorry, it doesn't work for variables. Should work if show_table_status_replacement_query will be a static private member of MySQLHandler.
```
// In MySQLHandler.h
class MySQLHandler
{
...
private:
    static const String MySQLHandler::show_table_status_replacement_query;
}

// At the end of MySQLHandler.cpp
const String MySQLHandler::show_table_status_replacement_query("SELECT"
```

Reply:
seems like it should be 
`
private:
    static const String show_table_status_replacement_query;
`
, or during build following error occurs:
`../dbms/programs/server/MySQLHandler.h:63:25: error: extra qualification 'DB::MySQLHandler::' on member 'show_table_status_replacement_query' [-fpermissive]
   63 |     static const String MySQLHandler::show_table_status_replacement_query;
      |                         ^~~~~~~~~~~~
ninja: build stopped: subcommand failed.`